developers/in-depth-tutorial-insurance-claims-arbitration.md

```diff
@@ -0,0 +1,387 @@
+
+### Insurance Arbitrator Contract
+
+This smart contract allows insurers to issue insurance policies by depositing insured amount, designating the insured beneficiary and describing insured event.
+
+At any time anyone can submit a claim that the insured event has occurred by posting oracle bonding. Insurance Arbitrator resolves the claim through Optimistic Oracle by passing a templated question with insured event description in ancillary data using `YES_OR_NO_QUERY` price identifier as specified in [UMIP-107](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-107.md).
+
+If the claim is confirmed and settled through Optimistic Oracle this contract automatically pays out insurance coverage to the insured beneficiary. If the claim is rejected the policy continues to be active ready for the subsequent claim attempts.
+
+### Development environment and tests
+
+#### Install dependencies
+
+To install dependencies, you will need to install the long-term support version of nodejs, currently nodejs v16, and yarn. You can then install dependencies by running yarn with no arguments from the root of [developer's quick-start repo](https://github.com/UMAprotocol/dev-quickstart):
+
+```bash
+yarn
+```
+
+#### Compiling your contracts
+
+We will need to run the following command to compile the contracts and make the typescript interfaces so that they are easy to work with:
+
+```bash
+yarn hardhat compile
+```
+
+### Contract implementation
+
+#### Contract creation and initialization
+
+`_finderAddress` parameter in the constructor points the Insurance Arbitrator to the entry point of the rest of UMA environment. This can be either fetched from the relevant [networks](https://github.com/UMAprotocol/protocol/tree/master/packages/core/networks) file looking up the address of deployed `Finder` contract or you can provide your own `Finder` instance if deploying UMA [protocol](https://github.com/UMAprotocol/protocol) in your own sandboxed environment.
+
+`_currency` parameter in the constructor allows choosing the settlement and bonding currency for insurance claims. This should be approved as whitelisted UMA collateral. Please check [Approved Collateral Types](../resources/approved-collateral-types.md) for production networks or call `getWhitelist()` for any of test networks to the [Address Whitelist](https://github.com/UMAprotocol/protocol/blob/master/packages/core/contracts/common/implementation/AddressWhitelist.sol) contract. Alternatively, one can approve the token with `addToWhitelist` method to the Address Whitelist contract if working in a sandboxed UMA environment.
+
+`_timerAddress` is used only when running unit tests locally to simulate advancement of time. For all the public networks (including testnets) zero address should be used.
```

Comment:
```suggestion
`_timer` is used only when running unit tests locally to simulate the advancement of time. For all the public networks (including testnets) zero address should be used.
```

Reply:
the three changes below are to match with the implementation changes made in this PR: https://github.com/UMAprotocol/dev-quickstart/pull/18 that I did to clean this up a little bit.